lib/models/work-item.js

```diff
@@ -144,13 +149,19 @@ function WorkItemModelFactory (Model, _, Promise, assert, Constants, configurati
             });
         },
 
-        setFailed: function setFailed(leaseToken, workItems) {
+        setFailed: function setFailed(leaseToken, alertMessage, workItems) {
```

Comment:
Thanks for the comments. I put alertMessage to make it convenient to slice possible workItems array.
I already update all setFailed so suggest to keep as it is.